actions/setup-conftest/action.yaml

```
@@ -18,6 +18,14 @@ runs:
         path: ${{ github.workspace }}/bin/conftest
         key: conftest-${{ runner.os }}-${{ runner.arch }}-${{ inputs.version }}
 
+    - name: Normalize runner.os to match Conftest release
+      shell: sh
+      env:
+        OS: ${{ runner.os }}
+      run: |
+        [[ "$OS" == 'macOs' ]] && OS='Darwin'
```

(if you keep this approach)

`[[` is a bashism

```suggestion
        [ "$OS" == 'macOs' ] && OS='Darwin'
```